deploy/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
```

Comment:
this  yaml is not required in this repo, the rbac will be installed as a part of installing chaos-operator